k2/util/Logging.cc

```diff
@@ -0,0 +1,57 @@
+// $FILE_PATHNAME $HEADER_FILENAME $HEADER_PATHNAME
```

Comment:
please replace this line with the one used by other k2 files.

Reply:
Done